recipes/llvm-core/all/conanfile.py

```diff
@@ -0,0 +1,289 @@
+from conans.errors import ConanInvalidConfiguration
+from conans import ConanFile, CMake, tools
+
+from collections import defaultdict
+import json
+import re
+import os.path
+import os
+
+
+class LLVMCoreConan(ConanFile):
+    name = 'llvm-core'
+    description = (
+        'A toolkit for the construction of highly optimized compilers,'
+        'optimizers, and runtime environments.'
+    )
+    license = 'Apache-2.0 WITH LLVM-exception'
```

Comment:
As a remark for all reviewers (and my future self), this is actually the correct SPDX annotation :+1: